src/main/resources/reference-data/sscs-venues.csv

```diff
@@ -89,6 +89,7 @@
 199,"SC199","SSCS Cardiff","Plas Dolerw","Milford Road","","Newtown","Powys","SY16 2EH","",301,"https://www.google.com/maps/place/Newtown+SY16+2EH,+UK/@52.5154719,-3.3240503,18.59z/data=!4m5!3m4!1s0x486ffb6490d5c873:0x250622992707e85d!8m2!3d52.5156711!4d-3.323016","No","Newtown","Not active"
 200,"SC200","SSCS Cardiff","Devon & Cornwall Probation Area","St Catherine's House","5 Notte Street","Plymouth","Devon","PL1 2TS","01752 268361",4503,"https://www.google.com/maps/place/Devon+%26+Cornwall+Probation+Area/@50.3683877,-4.1444777,19.9z/data=!4m13!1m7!3m6!1s0x486c934e0e649539:0x5c5c14a015c134f8!2sNotte+St,+Plymouth+PL1+2TS,+UK!3b1!8m2!3d50.3684757!4d-4.1440661!3m4!1s0x486c934e0eddf545:0xe13156092c94bcb9!8m2!3d50.3684152!4d-4.1440538","Yes","Plymouth",""
 201,"SC201","SSCS Cardiff","Havant Magistrates Court Building","The Courthouse","Elmleigh Road","","Havant","PO9 2AL","02 392 839 238",4503,"https://www.google.com/maps/place/Magistrates+Court/@50.8565386,-0.9854132,17z/data=!3m1!4b1!4m5!3m4!1s0x487444db5abdb8a9:0xd441b0067d2e7b28!8m2!3d50.8565352!4d-0.9832245","Yes","Havant",""
+203, "SC203", "SSCS Cardiff", "Southampton Crown Court", "The Courts of Justice", "London Road", "Southampton", "Hants", "SO15 2XQ", "", "4503", "https://www.google.com/maps/place/Southampton+Crown+Court/@50.9136832,-1.4033626,17.45z/data=!3m1!5s0x487476ac23b8b3e5:0x9e7862fbbec9109e!4m13!1m7!3m6!1s0x487476ac3b782539:0x5548c2a84b80bd0b!2sSouthampton+SO15+2XQ!3b1!8m2!3d50.9136969!4d-1.4029679!3m4!1s0x487476ac3c7dec37:0xc7e2f12290ad7393!8m2!3d50.9138088!4d-1.4029186", "Yes", "Southampton", ""
```

Comment:
Remove the space before each quote to match with the rest of the file.

Reply:
Removed the spaces